.github/workflows/codepreview.yml

```diff
@@ -0,0 +1,63 @@
+name: Create preview environment
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+
+concurrency:
+  # The preview script can't handle concurrent deploys
+  group: codepreview
+  cancel-in-progress: false
+
+# TODO: Define minimal permissions, I haven't found which one is necessary to allow writing comments on commits
+# see https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs
+#permissions:
+#  contents: read # for checkout
+
+jobs:
+  preview:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install ansible
+        run: python3 -m pip install --user ansible
+
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Compile frontend
+        run: npm install && npm run build:milkomeda-c1-testnet
```

Comment:
Hi @AlexITC . The code preview is great!!

As you may have noticed, we have 2 "production" environments: one that gets deployed when we push to `milkomeda-c1-testnet` branch and one that gets deployed to `main` branch. Therefore, could you:

1. create two codepreview workflows.

2. set the current one (matching `main`) to run `build:milkomeda-c1`.

3. set the new one (matching `milkomeda-c1-testnet`) to run `build:milkomeda-c1-testnet`.

?

Reply:
Done.